src/server/server.ts

```diff
@@ -84,15 +84,18 @@ export async function startApplication(args: CLIArguments) {
         const manifestPath = 'demo/app.json';
         const manifestUrl = `http://localhost:${PORT}/${manifestPath}`;
 
-        const fetchRequest = await fetch(getProviderUrl(args.providerVersion)).catch((err: string) => {
+        const demoFetchRequest = await fetch(manifestUrl);
+        const serviceFetchRequest = await fetch(getProviderUrl(args.providerVersion)).catch((err: string) => {
             throw new Error(err);
         });
 
-        if (fetchRequest.status === 200) {
-            const providerManifestContent = await fetchRequest.json();
+        if (serviceFetchRequest.status === 200 && demoFetchRequest.status === 200) {
+            const demoManifest = await demoFetchRequest.json();
+            const providerManifestContent = await serviceFetchRequest.json();
             console.log('Launching application');
 
             connect({uuid: 'wrapper', manifestUrl}).then(async fin => {
+                const demo = fin.Application.wrapSync({uuid: demoManifest.startup_app.uuid, name: demoManifest.startup_app.name});
```

Comment:
What is this used for?

Reply:
To kill the demo app.

Reply:
Oh right I missed where it was actually being used but see it now